deploy/non-olm/v1.1.2/operator.yml

```diff
@@ -0,0 +1,261 @@
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - apps
+  resourceNames:
+  - mig-operator
+  resources:
+  - deployments/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - migration.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: migration-operator
+  namespace: "openshift-migration"
+subjects:
+- kind: ServiceAccount
+  name: migration-operator
+roleRef:
+  kind: Role
+  name: migration-operator
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: migration-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: migration-operator
+    namespace: "openshift-migration"
+namespace: "openshift-migration"
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: migration-operator
+  namespace: "openshift-migration"
+  labels:
+    app: migration
+spec:
+  selector:
+    matchLabels:
+      app: migration
+  template:
+    metadata:
+      labels:
+        app: migration
+    spec:
+      serviceAccountName: migration-operator
+      containers:
+      - name: ansible
+        command:
+        - /usr/local/bin/ao-logs
+        - /tmp/ansible-operator/runner
+        - stdout
+        image: quay.io/ocpmigrate/mig-operator:release-1.1
+        imagePullPolicy: Always
+        volumeMounts:
+        - mountPath: /tmp/ansible-operator/runner
+          name: runner
+          readOnly: true
+      - name: operator
+        image: quay.io/ocpmigrate/mig-operator:release-1.1
+        imagePullPolicy: Always
+        volumeMounts:
+        - mountPath: /tmp/ansible-operator/runner
+          name: runner
+        env:
+        - name: OPERATOR_NAME
+          value: migration-operator
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: REGISTRY
+          value: quay.io
+        - name: PROJECT
+          value: ocpmigrate
+        - name: MIG_CONTROLLER_REPO
+          value: mig-controller
+        - name: MIG_UI_REPO
+          value: mig-ui
+        - name: MIGRATION_REGISTRY_REPO
+          value: registry
+        - name: MIGRATION_REGISTRY_TAG
+          value: release_2.7
+        - name: VELERO_REPO
+          value: velero
+        - name: VELERO_PLUGIN_REPO
+          value: migration-plugin
+        - name: VELERO_RESTIC_RESTORE_HELPER_REPO
+          value: velero-restic-restore-helper
+        - name: VELERO_AWS_PLUGIN_REPO
+          value: velero-plugin-for-aws
+        - name: VELERO_GCP_PLUGIN_REPO
+          value: velero-plugin-for-gcp
+        - name: VELERO_AZURE_PLUGIN_REPO
+          value: velero-plugin-for-microsoft-azure
+        - name: VELERO_TAG
```

Comment:
Are all these tags correct? It's really confusing to keep these straight between versions.

Reply:
There shouldn't be any changes to these for 1.1.2.

Reply:
In any case, the repo tags probably won't change even with major versions, although when we increment fusor- tags to the next version, "fusor-1.2" for velero will become "konveyor-1.3"

Reply:
@sseago will that be when we upgrade to use velero 1.3?

Reply:
Yes. Until then, the existing tags/branches should still be valid -- at least assuming we build images with the same tags as the branch names in github, we have fusor-1.2 but not konveyor-1.2, and we will (eventually) have konveyor-1.3 but never konveyor-1.2 for velero.

Reply:
Ya, tags upstream will stay the same between z's.... this PR isn't even technially needed upstream to get the changes. Downstream though we will change SHAs so we'll need a version bump...